vm_helper_files/Makefile

```diff
@@ -102,7 +92,7 @@ install-release-online:
 		fi; \
 	fi; \
 
-install-release-offline:
+install-release-offline: install-tools-deps
```

Comment:
at what point do you go offline for testing this? This command is supposed to be ran offline, but `install-tools-deps` requires internet connectivity